packages/website/pages/[...slug].tsx

```diff
@@ -295,9 +295,12 @@ export const getStaticPaths: GetStaticPaths = async () => {
           ?.sectionCollection?.items[0];
     }
 
+    const sectionSlug = section?.slug ?? 'default-section';
+    const itemSlug = item?.slug ?? 'default-item';
```

Comment:
Do these exist?

Reply:
they usually do, but the type script error is now blocking the build, so i implemented a quick fallback to avoid the error.
the issue is that neither item nor section are properly typed and so this is the point where it falls trough. i can also go ahead and do a wider refactoring, fixing the initial types.

Reply:
I mean do these default sections and items exist or will something break if they actually got used?

Reply:
Ahh, no, just the slug name and item would be displayed like that.